aten/src/ATen/native/cuda/SortingKthValue.cu

```diff
@@ -241,7 +241,7 @@ std::tuple<Tensor&, Tensor&> kthvalue_out_cuda(
     bool keepdim) {
   auto result = [&]() {
     NoNamesGuard guard;
-    return kthvalue_out_impl_cuda(values, indices, self, k, dim, keepdim);
+    return kthvalue_out_impl_cuda(values, indices, self.contiguous(), k, dim, keepdim);
```

Comment:
Would you just add a comment here or above the kthvalue_out_impl_cuda() that the function expects (and only works properly on) contiguous tensors?